projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -344,7 +344,7 @@ private static function stats_array_to_string( $kvs ) {
 
 		// Encode into JSON object, and then encode it into a string that's safe to embed into Javascript.
 		// We will then use JSON.parse method in JS to read the array.
-		return wp_json_encode( wp_json_encode( $kvs ) );
+		return wp_json_encode( wp_json_encode( $kvs, JSON_UNESCAPED_SLASHES ) );
```

Comment:
This seems like another PHP 5.2-compat sort of thing. If you don't want to dig deeper into it, then at least do
```suggestion
		return wp_json_encode( wp_json_encode( $kvs, JSON_UNESCAPED_SLASHES ), JSON_UNESCAPED_SLASHES );
```
If you do want to dig deeper, you should be able to do
```suggestion
		return wp_json_encode( $kvs, JSON_UNESCAPED_SLASHES );
```
here and then change the code near line 184 to remove the `JSON.parse()` call.

Reply:
Tracing it down, it seems removing the double-encoding and the `JSON.parse()` should work, though I added the full set of flags: e2803b9247

I had to stop myself from moving the `wp_json_encode()` into `build_stats_details()` instead, which I think is a better location.

Reply:
Good catch, the full set of flags is needed.